specs/requests/invites_specs.py

```diff
@@ -0,0 +1,58 @@
+import sys
+
+from expects import expect, have_key, equal, contain
+from mamba import description, before, after, it
+from neomodel import db
+
+from specs.factories.team_factory import TeamFactory
+from specs.factories.user_factory import UserFactory
+from specs.spec_helpers import Context
+
+sys.path.insert(0, 'specs')

```

Comment:
What does this do?

Reply:
I don't know, but It's at the beginning of all our test files...

Reply:
sys.path is the path python uses to import modules. Inserting 'specs' at 0 makes the specs folder the first one to look for for modules to import. Python should however automatically add the current file and folder to the top of the list, so I don't think this is needed. You can try removing it and see if everything still works.

If not it'd be interesting to see how and why that affects imports.

Reply:
Always important to question older decisions. Either you improve something or we learn something :P